service/lib/agama/storage/config_checkers/boot.rb

```diff
@@ -55,7 +55,12 @@ def device_alias
         def missing_alias_issue
           return unless configure? && device_alias.nil?
 
-          error(_("There is no boot device alias"))
+          # Currently this situation only happens because the config solver was not able to find
+          # a device config containing a root volume. The message could become inaccurate if the
+          # solver logic changes.
+          error(
+            _("The boot device cannot be automatically selected because there is not root volume")
```

Comment:
I'm not sure whether "volume" means something in this context. If anything, it makes me think on LVM. I would better say "there is no root (/) file system"

Reply:
I have the very same doubt.